src/Microsoft.Health.Fhir.Stu3.Core.UnitTests/Features/Search/SearchOptionsFactoryTests.cs

```diff
@@ -222,6 +223,41 @@ public void GivenASearchParamWithEmptyValue_WhenCreated_ThenSearchParamShouldBeA
             Assert.Equal(queryParameters, options.UnsupportedSearchParams);
         }
 
+        [Fact]
```

Comment:
Seems like these tests could go into the shared project since it's not STU3 specific behavior.

Reply:
Moved the test cases into core along with other non R3 specific test cases.